bin/import_fisher.py

```diff
@@ -142,7 +142,7 @@ def _split_wav_and_sentences(data_dir, trans_data, original_data, converted_data
 
             print("splitting {} according to {}".format(wav_files, trans_file))
 
-            origAudios = [wave.open(wav_file, "r") for wav_file in wav_files]
+            origAudios = [librosa.load(wav_file, sr=None, mono=False) for wav_file in wav_files]
```

Comment:
A stereo WAV file here would be a bug, right?

Reply:
As far as I can tell yes. However, the original code, for whatever reason, handled stereo.